### PR TITLE
feat: Generate compose files for backward compatibility test

### LINKED ISSUE #693: [BCT] Generate compose files for backward compatibility test
# 🚀 Feature Request

### Relevant Package [**REQUIRED**]

 This feature request is for Backward compatibility Test

### Description [**REQUIRED**]
 

1. Deploy edgex with Jakarta release.
2. Run integration test.
3. Stop services
4. Redeploy services with device-services and app-services of Jakarta release and core-services of Levski release.
5. Run integration test.

Ref: https://github.com/edgexfoundry/edgex-taf-pipelines/issues/108

### Describe the solution you'd like
 If you have a solution in mind, please describe it.

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

### PR DESCRIPTION
**Pull request by cherrycl**
1. Generate compose files for backward compatibility test
2. Set tag backward-skip on tests created after kamakura
3. Remove duplicate keyword
4. Exception handling for http server script
5. Remove unused files.

Fix #693 

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [X] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: TAF/testCaseModules/keywords/common/commonKeywords.robot [hunk 1]
```diff
@@ -308,10 +308,12 @@ No ${metrics_name} Found In File
...
-    Start process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-subscriber.py ${topic} ${keyword} arg &
-    ...            shell=True  stdout=${WORK_DIR}/TAF/testArtifacts/logs/mqtt-subscriber-${current_time}.log
-    ...            stderr=${WORK_DIR}/TAF/testArtifacts/logs/mqtt-error-${current_time}.log
     Set Test Variable  ${subscriber_file}  mqtt-subscriber-${current_time}.log
+    Set Test Variable  ${error_file}  mqtt-error-${current_time}.log
+    ${handle}  Start process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-subscriber.py ${topic} ${keyword} arg &
+    ...            shell=True  stdout=${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}
+    ...            stderr=${WORK_DIR}/TAF/testArtifacts/logs/${error_file}
     sleep  1s
+    [Return]  ${handle}
```
Comment (jinlinGuan) [inline TAF/testCaseModules/keywords/common/commonKeywords.robot:319]: Is this necessary? We should use a consistent format for this keyword in all the testcases. 
Either `Run MQTT Subscriber Progress And Output <args>` or `${handle} Run MQTT Subscriber Progress And Output  <args>`

Comment (cherrycl) [reply TAF/testCaseModules/keywords/common/commonKeywords.robot:319]: Return the parameter, can kill the single process.
Update keywords.

File: TAF/testScenarios/integrationTest/UC_mqtt_message_bus/core_data_config.robot [hunk 2]
```diff
@@ -43,8 +41,7 @@ CoreConfig002 - Set core-data MessageQueue.SubscribeTopic not match device-virtu
                 ...           AND  Set MessageQueue SubscribeTopic=edgex/events/device/# For core-data On Consul
 
 CoreConfig003 - Customize core-data MessageQueue.PublishTopicPrefix
-    Given Start process  python ${WORK_DIR}/TAF/utils/src/setup/mqtt-subscriber.py eedgex/events/custom/# CorrelationID arg &   # Process for MQTT Subscriber
-    ...                shell=True  stdout=${WORK_DIR}/TAF/testArtifacts/logs/mqtt-subscriber.log  
+    Given Run MQTT Subscriber Progress And Output  eedgex/events/custom/#
```
[the remark this answers is not in the document]
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_mqtt_message_bus/core_data_config.robot]: Updated.

**Review by jumpingliu — APPROVED** [review]
LGTM

**Review by jinlinGuan — COMMENTED** [review]
LGTM

**Pull request merged**